src/main/java/tree/ImpTargetTreeNode.kt

```diff
@@ -1,5 +1,7 @@
 package tree
 
+data class MatchedResponse(val matchedString: String, val currentIndex: Int)
```

Comment:
I had to create this data class, however I'm still not sure about its placement. Should this be here or where should I put it?

Reply:
No here is good, because no other class uses this data class.